packages/react/src/patterns/sub-patterns/Quote/Quote.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ArrowRight20, Launch20 } from '@carbon/icons-react';
+import {
+  settings as ddsSettings,
+  decodeString,
+} from '@carbon/ibmdotcom-utilities';
+import { HorizontalRule, LinkWithIcon } from '@carbon/ibmdotcom-react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Quote Pattern
+ *
+ * @param {object} props props Object
+ * @param {string} props.markType type of the quote marks
+ * @param {string} props.copy quote copy
+ * @param {object} props.source source object
+ * @param {object} props.source.heading name heading for quote source
```

Comment:
@RaphaelAmadeu can you check the types here? I don't think all are objects...

Reply:
@jeffchew I've updated this in the latest commit, Thanks!